src/plugins/template/backend/ops/grid_sample.cpp

```diff
@@ -34,23 +64,17 @@ template <>
 bool evaluate_node<ov::op::v9::GridSample>(std::shared_ptr<ov::Node> node,
                                            ov::TensorVector& outputs,
                                            const ov::TensorVector& inputs) {
-    auto element_type = node->get_output_element_type(0);
-    if (ov::is_type<ov::op::v1::Select>(node) || ov::is_type<ov::op::util::BinaryElementwiseComparison>(node))
-        element_type = node->get_input_element_type(1);
```

Comment:
Why the change not required anymore?

Reply:
Is not required as this evaluate is registered for `v9::GridSample` and this check will be false.